testsuite/WIKI.md

```diff
@@ -66,35 +70,37 @@ For example, a successful test might look like the following
 ```
 
 ### Coverage ###
-We test code coverage of the unit tests with the  [coverage](http://nedbatchelder.com/code/modules/rees-coverage.html) plugin of nose. Currently, this is done automatically as part of the Travis job on Python 2.7 and is viewable on **[coveralls](https://coveralls.io/r/MDAnalysis/mdanalysis?branch=develop)**.
+We test code coverage of the unit tests with the  [coverage](http://nedbatchelder.com/code/modules/rees-coverage.html) plugin of nose. Currently, this is done automatically as part of the Travis job and is viewable on **[coveralls](https://coveralls.io/r/MDAnalysis/mdanalysis?branch=develop)**.
+
+If you want to generate a coverage report manually, you will need pytest-cov installed.
+`pip install pytest-cov`
 
-If you want to generate a coverage report manually you can run
+Then you can run -
 ```
 cd testsuite
 pytest --numprocesses 4 --cov=MDAnalysis --cov-report html
 ```
 
 # Details #
-We are borrowing some of NumPy's testing frame work; thus, numpy **must** be installed for the tests to run at all. The tests require at least numpy 1.5.
+We are borrowing some of NumPy's testing frame work; thus, numpy **must** be installed for the tests to run at all.
 
 ## Running tests from within python ##
 Run all the tests with
 ```
-   import MDAnalysis.tests
-   MDAnalysis.tests.test()
+python -c 'from MDAnalysisTests import run; run()'
 ```
 
 ## Running tests from the command line ##
-Instead of running tests from within python, one can also run them via the `mda_nosetests` script that ships with MDAnalysisTests. With version 0.11.0 the test subsystem was overhauled to allow the incorporation of customized nose plugins. In order for them to work tests must be invoked via our own wrapper function `MDAnalysis.tests.test()`. This is what the `mda_nosetests` script does for you. Alternatively, you can call `MDAnalysis.tests.test()` from the interpreter. `mda_nosetests` strives to be compatible and interchangeable with nose's [nosetests](http://somethingaboutorange.com/mrl/projects/nose/0.11.2/usage.html) script, with [added functionality](#compatibility-with-nosetests).
-
+Instead of running tests from within python, one can also run them via the command line.
 Go into the tests directory (or the package root)
 ```
 cd /testsuite/MDAnalysisTests
 ```
-and invoke ./mda_nosetests directly to run **all tests** on two processors in parallel ("`%`" is the shell prompt and should _not_ be typed):
+and invoke pytest directly to run **all tests** on four processors in parallel ("`%`" is the shell prompt and should _not_ be typed):
 ```
 % pytest --numprocesses 4
 ```
+Again, to run the tests in parallel (use `--numprocesses 4`) you need to install `pytest-xdist` via `pip install pytest-xdist`
```

Comment:
Don't say how to install xdist. For people using conda the correct command would be `conda install pytest`. Others might want to use package manager from the distribution they are using.